README.md

```diff
@@ -130,6 +130,7 @@ Real-time electricity data is obtained using [parsers](https://github.com/tmrowc
 - Ukraine: [UKRENERGO](https://ua.energy/activity/dispatch-information/ues-operation/)
 - United States: [PYISO](https://github.com/WattTime/pyiso)
 - United States (California): [CAISO](http://www.caiso.com/Pages/default.aspx)
+- United States  (MISO):[MISO](https://api.misoenergy.org/MISORTWDDataBroker/DataBrokerServices.asmx?messageType=getfuelmix&returnType=json)
```

Comment:
Maybe we should do one level of indent for the US (as done in capacities?

Reply:
Sure will do.

Reply:
I'll also get rid of the pyiso reference.